src/pair.c

```diff
@@ -551,111 +551,156 @@ int get_pair_osc_arg_index(char* varname, char* oscargs, uint8_t argc, uint8_t s
     return i;
 }
 
+static int is_ws(const char *s)
+{
+    while(*s && *s == ' ') s++;
```

Comment:
Yeah, you're right. I forgot that ' ' in an sscanf template can also match \t (and possibly a few other whitespace characters such as \f and \v). Replacing the comparison with isspace() should fix this.